lib/event_sourcery/event_processing/esp_runner.rb

```diff
@@ -106,6 +106,7 @@ def send_signal_to_remaining_processes(signal)
       def record_terminated_processes
         until all_processes_terminated? || (pid, status = Process.wait2(-1, Process::WNOHANG)).nil?
           event_processor = @pids.delete(pid)
+          next unless event_processor
```

Comment:
Is it worth logging a message in this case (with the PID)?

Reply:
Yeah, good idea.

Reply:
Added in c5b10834611115c5389b8cb5b4a554d7efd59891